src/main/java/it/cnr/isti/hpc/wikipedia/reader/WikipediaArticleReader.java

```diff
@@ -154,22 +156,19 @@ public void process(WikiArticle page, Siteinfo si) {
 				return;
 			}
 
-			if(page.isHelp()){
+			if(integerNamespace.equals(NamespaceCode.HELP_NAMESPACE_KEY.code)){
```

Comment:
Can we delete the `WikiArticle.isHelp` method then? Same question for changes below.

Reply:
@stathischaritos said this method is gone in the new version of bliki and that's precisely the reason of the change above.